pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2110,6 +2165,50 @@ static __always_inline unsigned short get_inode_mode_from_fd(u64 fd)
     return READ_KERN(f_inode->i_mode);
 }
 
+static __always_inline void detect_hooked_syscalls(event_data_t *data){
```

Comment:
This function defiantly not detecting hooked syscalls, so this name is incorrect.
I suggest calling it `invoke_syscalls_addresses_event` which describes better the purpose here.